ISSUE #606: Refactor footer to include more information

alexanderturinske opened the pull request:
Closes #606

richardxia commented on app/styles/components/_footer.scss:
```diff
@@ -13,46 +13,61 @@
 
+  .footer-service-guide {
+    text-align: center;
+    font-weight: 1000;
```
Might be safer to use 900. I didn't actually realize that 1000 is a supported value now, but older browsers might only support 100-900. This chart from MDN doesn't even know if Edge or Safari support 1000: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Browser_compatibility

alexanderturinske replied:
Done

richardxia commented on app/components/ui/Footer.jsx:
```diff
@@ -5,14 +5,40 @@ function Footer() {
+          </section>
+          <section className="footer-links">
+            <ul>
+              <li>ABOUT US</li>
```
Nit: My personal approach for this kind of stuff is to type out the text in the source code using normal casing and adding a CSS rule to set `text-transform: uppercase`. Also, I would probably make this first thing a heading tag like `<h1>/<h2>/...`, since it's more of a heading than a list item like the rest of the things.

alexanderturinske commented:
- Update class names to be BEM style
- Added footer icon
- updated for comments

Desktop
<img width="1373" alt="screen shot 2019-01-06 at 12 36 30 am" src="https://user-images.githubusercontent.com/12485238/50733954-9d40cd80-114b-11e9-9c38-e5c53e7d5a0a.png">
Mobile
<img width="632" alt="screen shot 2019-01-06 at 12 36 42 am" src="https://user-images.githubusercontent.com/12485238/50733955-9d40cd80-114b-11e9-9979-f4ae9b6d13c6.png">

lexholden commented on app/components/ui/Footer.jsx:
```diff
@@ -5,14 +5,40 @@ function Footer() {
+              <li>LEGAL</li>
+              <li><Link to="/terms-of-service">Terms of Service</Link></li>
+              <li><Link to="/privacy-policy">Privacy Policy</Link></li>
+              <li><a href="#">API Policy</a></li>
```
I don't think it exists yet, I added the content for the two above a while back, but those pages didn't exist either at the time. I guess you need to add a new page in `app/components/pages/legal`

lexholden commented on app/components/ui/Footer.jsx:
```diff
@@ -5,14 +5,40 @@ function Footer() {
+            <ul>
+              <li>HELP</li>
+              <li><a href="#">FAQ</a></li>
+              <li><a href="#">Digital Literacy</a></li>
```
Same for these, though @derekfidler I guess the specs for this links is a direct request from the city? We need content for them otherwise we should comment the links out until we can add the pages.

alexanderturinske replied:
@richardxia @lexholden I will comment out these links until we know where they go. That way the commit can be unblocked.

alexanderturinske commented:
Commented out links with no well defined redirects
Desktop
![image](https://user-images.githubusercontent.com/12485238/50742906-f4838400-11c5-11e9-8000-64d6f9ced32f.png)
Mobile
![image](https://user-images.githubusercontent.com/12485238/50742909-036a3680-11c6-11e9-968d-5a537be0667d.png)

richardxia commented on app/components/ui/Footer.jsx:
```diff
@@ -4,15 +4,44 @@ import { Link } from 'react-router';
+              <h1>About Us</h1>
+              <li><a href="sheltertech.org">Our Story</a></li>
+              <li><a href="sheltertech.org/volunteer">Volunteer</a></li>
+              <li><a href="sheltertech.org/donate">Donate</a></li>
```
`/donate` doesn't appear to be a real URL.

alexanderturinske replied:
Ah, I will comment it out.

alexanderturinske replied:
Done

richardxia commented on app/utils/config.example.js:
```diff
@@ -0,0 +1,7 @@
+
```
I don't think you meant to commit this file.

alexanderturinske replied:
Curses; that is the second time that has gotten pushed up. WIll remove.

richardxia commented on app/components/ui/Footer.jsx:
```diff
@@ -4,15 +4,44 @@ import { Link } from 'react-router';
+          <section className="service-guide">
+            <div className="service-guide__icon" />
+            <h1 className="service-guide__text">
+              SF Service Guide
```
I think we want this to conditionally say "Ask Darcel" or "SF Service Guide" depending on what domain name the site is being served from. Here's an example of where we're currently doing a check based on the subdomain: https://github.com/ShelterTechSF/askdarcel-web/blob/861898cbc87b5e6459791b0ecbb3b6cf1cbea791/app/pages/HomePage.jsx#L43

alexanderturinske replied:
Will do

alexanderturinske commented on app/styles/components/_footer.scss:
```diff
@@ -13,49 +13,77 @@
+    display: block;
+    padding-left: 0;
+    text-align: center;
+    padding-left: 0;
```
There are two `padding-left`; why?

alexanderturinske replied:
Deleted
